app/helpers/blacklight_config_helper.rb

```diff
@@ -10,7 +10,14 @@ def self.add_common_default_solr_params_to_config!(config)
       'q.alt': '*:*',
       defType: 'dismax',
       qf: %(
-        sw_display_title_tesim^5
+        main_title_text_anchored_im^100
+        main_title_text_unstemmed_im^50
+        main_title_tenim^10
+        full_title_unstemmed_im^10
+        full_title_tenim^5
+        additional_titles_unstemmed_im^5
+        additional_titles_tenim^3
+
```

Comment:
these changes and those to sul_solr_configs are what will change the relevancy ranked results.